benchmarks/unbounded-cardinalities/README.md

```diff
@@ -0,0 +1,30 @@
+# Unbounded Cardinalities Benchmarks
```

Comment:
Nice! We should add the literature references for the racks and house benchmark below